advisor/src/test/kotlin/advisors/VulnerableCodeTest.kt

```diff
@@ -103,8 +104,7 @@ class VulnerableCodeTest : WordSpec({
             langResults[0].advisor.name shouldBe ADVISOR_NAME
             val expLangVulnerability = Vulnerability(
                 id = "CVE-2014-8242",
-                severity = 42.0f,
-                url = URI("http://localhost:8000/api/vulnerabilities/v1/")
+                createReferences(severity = 42.0f, url = URI("http://localhost:8000/api/vulnerabilities/v1/"))
```

Comment:
How about always making the speaking `URI()` argument first, and only use a named parameter for the second `severity =`, like

    createReferences(URI("http://localhost:8000/api/vulnerabilities/v1/"), severity = 42.0f)

Reply:
Similar below.